packages/suite/src/components/suite/OverflowItems.tsx

```diff
@@ -0,0 +1,130 @@
+import { ReactNode, useCallback, useEffect, useRef, useState } from 'react';
+import { usePrevious, useUpdateEffect } from 'react-use';
+import { useSelector } from 'src/hooks/suite';
+import styled from 'styled-components';
+
+export type OverflowItem = {
+    removeOrder?: number;
+    [key: string]: any;
+};
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-wrap: nowrap;
+    min-width: 0;
+`;
+
+interface OverflowItemsProps<T> {
+    className?: string;
+    items: T[];
+    visibleItemRenderer: (item: T, index: number) => ReactNode;
+    overflowRenderer: (items: T[]) => ReactNode;
+    minVisibleItems?: number;
+    alwaysRenderOverflow?: boolean;
+}
+
+export function OverflowItems<T extends OverflowItem>({
+    items,
+    className,
+    minVisibleItems = 0,
+    alwaysRenderOverflow = false,
```

Comment:
nit: `false` is not really needed since it's falsy when `undefined`

Reply:
https://github.com/trezor/trezor-suite/pull/9722/commits/9dacbbb1e944ac9b4042d3097509aff4a2207a08 fixed